ethereum/script/Deployment.s.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+pragma solidity ^0.8.30;
+
+import {Script, console} from "forge-std/Script.sol";
+import {Base} from "test/Base.sol";
+
+contract DeploymentScript is Script, Base {
+    function setUp() public {}
+
+    function run() public {
+        deployBridgeFromEnvironment();
```

Comment:
add here comment on which envs should be set and why

Reply:
we have `.env.example` and `README` for that